app/views/hiring_staff/organisations/show.html.haml

```diff
@@ -12,9 +12,9 @@
   - if @multiple_organisations && @organisation.is_a?(School)
     .govuk-grid-column-full
       - if AuthenticationFallback.enabled?
-        = link_to t('sign_in.organisation.change'), auth_email_change_organisation_path, class: 'govuk-link'
+        = link_to t('hiring_staff.organisations.change'), auth_email_change_organisation_path, class: 'govuk-link'
```

Comment:
Maybe we could start using: `govuk_link_to('Home', '/')` ?

https://dfe-digital.github.io/govuk-components/

Reply:
Yes definitely 👍